hasura.planx.uk/migrations/1725873202035_alter_table_public_teams_drop_column_submission_email/down.sql

```diff
@@ -0,0 +1,31 @@
+alter table "public"."teams" add column "submission_email" text;
+alter table "public"."teams" alter column "submission_email" drop not null;
```

Comment:
This line seems redundant - we're not defining a `NOT NULL` constraint when adding the column here.